hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/__tests__/vitest.setup.ts

```diff
@@ -16,7 +16,13 @@
 */
 
 import '@testing-library/jest-dom/vitest';
+import 'vitest-canvas-mock';
```

Comment:
Any canvas API we need to mock? Any use of this?

Reply:
So in the latest commit I had actually removed the canvas mock as I added manual mocking for the ECharts canvas.
Ref: https://github.com/apache/ozone/blob/540b413ba8cc433bb83f924f9c7425ee217c55ac/hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/__tests__/Overview.test.tsx#L50

Reply:
I see, seems I was looking at the outdated change.